### ISSUE TITLE
setlist.fm API changes (0.1 to 1.0)

### ISSUE THREAD

**Issue by SHOEGAZEssb** [gshackles/SetlistFmApi#1]
From a recent EMail I got:

Hey setlist.fm API users, 

We have recently deployed a new version of our API: https://api.setlist.fm/. This API is now the recommended way to access setlist.fm data and we’ll be retiring the old version of the api (/0.1) by the end of the year.

The migration from 0.1 to 1.0 is pretty straight forward as the endpoints have been kept very similar to the previous version. The same applies to the format where only minor adaptations are necessary. In order to use the new API version it is now necessary to provide your API key (can be found on https://www.setlist.fm/settings/api/) as HTTP header “x-api-key” in your request. To have a look at the new methods and format you can easily use the interactive web interface: https://api.setlist.fm/docs/1.0/ui/index.html

If you’re having troubles with the migration, please post your problems to the API forum http://www.setlist.fm/forum/setlistfm/setlistfm-api. Should you be having troubles with your API key or problems that can’t be posted publicly, please get in touch with us using one of the contact e-mail addresses or by replying to this e-mail.

tl;dr: There’s a new API version, please be sure to migrate by the end of the year or you won’t have access anymore.

cheers,
the setlist.fm team
